docker-compose.yml

```diff
@@ -7,8 +7,6 @@ services:
       - POSTGRES_DB=thenewboston_node
       - POSTGRES_USER=thenewboston
       - POSTGRES_PASSWORD=thenewboston
-    ports:
```

Comment:
This is used here to run PostgreSQL for local dev env:

```
.PHONY: up-dependencies-only
up-dependencies-only:
	docker-compose up --force-recreate db
```

Reply:
It conflicts with non containerized Postgres. Why do we need to expose it?

Reply:
I use containerized PostgreSQL on all projects I work on. Also `INSTALL.rst` does not suppose installation of PostgreSQL locally, just using it dockerized